src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+    </Step>
+    <Step>
+
+### Triage your cluster
+
+The Kubernetes overview dashboard shows your high level data about your cluster. You can find general data such as the count of pods and services, but more importantly you can find data about the health of your cluster such as the percent of pods running, the count of failed pods, the number of container restarts, and more. Be sure to scroll down to see all the graphs available to you.
+
+<img
+    title="tutorialOverviewdashboard"
+    alt="The main overview dashboard for the kubernetes capability"
+    src={tutorialOverviewdashboard}
+/>
+
+Use this dashboard to gauge the general health of your cluster. Here are a few things to look for:
+
+* Red or yellow tiles: think of yellow tiles as warnings to keep an eye on what it monitors. For example, if you have 2 unhealthy deployments you should take note and possibly plan to troubleshoot those deployments. Think of red tiles as critical alerts. These aren't necessarily failures in your system, but you should prioritize addressing them as immediately as possible.  
+* Anomalous spikes in graphs: there are various graphs that show things such as pending pods over time or even memory utilization over time. Spike are fairly normal, such as the spikes in the **Kubernetes Warning Events by Reason** graph in the screenshot above. These spikes happen regularly about every 5 minutes so they are no cause for concern. Look for spikes that happen outside of regular patterns or spike in a much larger magnitude than normal. 
+* Node readiness: observe whether nodes in the cluster are ready and able to host pods. Ensure that your cluster's infrastructure is prepared to handle workloads without any bottlenecks.
+* Resource count insights: keep a close eye on the number of pods, containers, nodes, or other Kubernetes resources within the cluster. While you won't always find something actionable, monitoring resource utilization allows you to plan for future scaling.
+
+Use the time selector in the top left of the page to see your data across time ranges to verify any troubling data isn't just random or to triage across a longer timeframe. 
+    </Step>
+</Steps>
+
+## Understand and monitor the orchestrated layer
+
+Continuing with our bakery analogy, the orchestrated layer is the baking stations within the bakery. Each station houses a set of ovens that work together to fulfill specific tasks. Each station might be designated to have three ovens running at all times. If an oven fails, the station will fire up a new oven to keep up with expectations.
+
+In a Kubernetes system, deployments have various specifications. For example, a deplyoment might be specified to always run four instances of an application in individual containers. The deployment will spin up pods containing that application until it meets that quota. If a pod were to fail, it would spin up a new one to continue to meet the specified  .
```

Comment:
Extra space at the end of this paragraph it looks like. I'm also seeing some more complex words and phrases in this and the previous section that you should consider simplifying: "various specifications" "resource utilization", things like that.